juju/juju
# [JUJU-395] Adds support for dashboard proxying on to CAAS.

This change allows users to proxy onto the Juju Dashboard in Juju3 for
both CAAS and IAAS models.

## Checklist


## QA steps

You will need to get the Juju Dashboard charm from here https://github.com/canonical-web-and-design/juju-dashboard-charm

Build the charm for both machine and kubernetes.

Bootstrap to microk8s and lxd and deploy the charm on both controllers. Once this is done relate the juju-dashboard to the controller. NOTE: These must be in the same model.

Run `juju dashboard` for both controllers and make sure you can get to the dashboard with the displayed credentials.

## Documentation changes

N/A

## Bug reference
N/A
